handsontable/src/core.js

```diff
@@ -4374,4 +4374,8 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
   });
 
   Hooks.getSingleton().run(instance, 'construct');
+
+  if (this.isRtl() && this.getSettings().fixedColumnsLeft) {
```

Comment:
I'm trying to do this, and It looks to work, however it is not perfect.

MetaManager is instantiated on the top of `Core` constructor. And the `instance` is put there as a `hot` parameter. That should provide access to full Core's API, however, it will not happen. Because, Core class methods are defined inside its constructor, after MetaManager.

I moved the required methods upper, to make `isRtl` accessible in a Mod. This makes it a lot messy, and some tests relating to collapsible columns stopped working. 

see 601227c